containers/roles/foreman/files/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -x
+
+/usr/bin/wait_on_postgres.py
+
+foreman-rake db:migrate
+echo $SEED_ADMIN_PASSWORD
+SEED_ADMIN_PASSWORD=$SEED_ADMIN_PASSWORD foreman-rake db:seed
+foreman-rake apipie:cache:index
```

Comment:
I recall this step being very slow. Future work should have a look at improving this.